isis/cmake/updateBuildPlugins.sh.in

```diff
@@ -0,0 +1,7 @@
+> ${CMAKE_BINARY_DIR}/lib/Camera.plugin
```

Comment:
Does this empty the file?

Reply:
@jessemapel yes that's what this is doing